editor/plugins/script_editor_plugin.cpp

```diff
@@ -1012,6 +1013,52 @@ void ScriptEditor::_menu_option(int p_option) {
 			file_dialog->set_title(TTR("Open File"));
 			return;
 		} break;
+		case FILE_REOPEN_CLOSED: {
+
+			if (previous_scripts.empty())
+				return;
+
+			String path = previous_scripts.back()->get();
+			previous_scripts.pop_back();
+
+			List<String> extensions;
+			ResourceLoader::get_recognized_extensions_for_type("Script", &extensions);
+			bool built_in = !path.is_resource_file();
+
+			if (extensions.find(path.get_extension()) || built_in) {
+				if (built_in) {
+					String scene_path = path.get_slice("::", 0);
+					if (!EditorNode::get_singleton()->is_scene_open(scene_path)) {
+						EditorNode::get_singleton()->load_scene(scene_path);
+						script_editor->call_deferred("_menu_option", p_option);
+						previous_scripts.push_back(path); //repeat the operation
+						return;
+					}
+				}
+
+				Ref<Script> scr = ResourceLoader::load(path);
+				if (!scr.is_valid()) {
+					editor->show_warning(TTR("Error: could not load file."), TTR("Error!"));
+					file_dialog_option = -1;
+					return;
+				}
+
+				edit(scr);
+				file_dialog_option = -1;
+				return;
+			} else {
+				Error error;
+				Ref<TextFile> text_file = _load_text_file(path, &error);
+				if (error != OK)
+					editor->show_warning(TTR("Error could not load file."), TTR("Error!"));
```

Comment:
Please use the same error string as above to avoid duplication of the content to translate (and have things consistent). It would be good to mention the file path to for the error to be more explicit.

You can do that with:
```
TTR("Could not load file at:") + "\n\n" + path
```

Alternatively:
```
vformat(TTR("Could not load file at '%s'."), path)
```
but if the path is long I guess it might make the dialog a bit awkward as it doesn't autowrap IINM.

Reply:
I actually copied the code from `FILE_OPEN` option (but probably made a mistake and it ended in different messages). Should I also change the old ones?
https://github.com/godotengine/godot/blob/3a6102a6f716904bb158819146e3fed7590b8106/editor/plugins/script_editor_plugin.cpp#L910
https://github.com/godotengine/godot/blob/3a6102a6f716904bb158819146e3fed7590b8106/editor/plugins/script_editor_plugin.cpp#L923

Reply:
Yes, please do.

Reply:
Done. Went with the first option.